### ISSUE TITLE
Explore storage visualization, preferences, and cleaning

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1554]
See example ![image](https://github.com/damus-io/damus/assets/64376233/5ea16774-2e37-415f-aa73-c8865cc92ad9)

**Comment by jb55** [comment]
On Fri, Sep 22, 2023 at 10:43:50AM -0700, alltheseas wrote:
>See example ![image](https://github.com/damus-io/damus/assets/64376233/5ea16774-2e37-415f-aa73-c8865cc92ad9)

yeah this is a good idea

**Comment by alltheseas** [comment]
@robagreda 👀

**Comment by alltheseas** [comment]
https://damus.io/note18na96y0mkzx7sjggqxtwzvvyhz25l3get3e8ly0mqlxggn42ay0qcjggdc

**Comment by alltheseas** [comment]
It might make sense to give the advanced user fine grained control over allocating max space to media, and to notes.

Details on local note search

https://damus.io/note17nlx3d304rwluytd0kp3wylzynnr3vfm9hxml9gfx0z45cg9fxesv5d3rf

**Comment by alltheseas** [comment]
https://damus.io/note1z03j56nnkvp5qvs3altcfx4fsptx5q8ylmfzynezfvr00aqavtcsvm43ew

> damus cache went back up to 10GB in less than 2 hours after clearing it, makes it rather tedious to clear, as there's no warning/notification about it, until i find my phone becoming immensely slow to use

**Comment by alltheseas** [comment]
> https://damus.io/note1z03j56nnkvp5qvs3altcfx4fsptx5q8ylmfzynezfvr00aqavtcsvm43ew
> 
> > damus cache went back up to 10GB in less than 2 hours after clearing it, makes it rather tedious to clear, as there's no warning/notification about it, until i find my phone becoming immensely slow to use

> i cleared 5 min ago, it's now on 1,63 GB

@danieldaquino earlier we had discussed setting more aggressive cache clear values. What are the downsides of halving the cache clearing interval? More data usage?

**Comment by danieldaquino** [comment]
> @danieldaquino earlier we had discussed setting more aggressive cache clear values. What are the downsides of halving the cache clearing interval? More data usage?

@alltheseas yes, it is mostly a tradeoff between data plan usage and storage usage

**Comment by jb55** [comment]
On Fri, Jan 12, 2024 at 01:38:21PM -0800, Daniel D’Aquino wrote:
>> @danieldaquino earlier we had discussed setting more aggressive cache clear values. What are the downsides of halving the cache clearing interval? More data usage?
>
>@alltheseas yes, it is mostly a tradeoff between data plan usage and storage usage

Maybe we can look into a feature in the future where we set a maximum
storage quota. Telegram has something like this.

**Comment by alltheseas** [comment]
> Maybe we can look into a feature in the future where we set a maximum storage quota. Telegram has something like this.

Yes, and this would be complemented with low data mode.

**Comment by alltheseas** [comment]
Reported clear cache results: 18 GB reduced to 16 GB

https://damus.io/nevent1qqsp9dv3aukwjgcn9t97qyk53qumsl7n0ugmvww8sted6uhdg0dclxqprpmhxue69uhhyetvv9ujumn0wdmksetjv5hxxmmdqyxhwumn8ghj7mn0wvhxcmmvqy2hwumn8ghj7etyv4hzumn0wd68ytnvv9hxgqg5waehxw309aex2mrp0yhxgctdw4eju6t0jvp9zm

**Comment by alltheseas** [comment]
prospective customer feedback

![image](https://github.com/user-attachments/assets/46bf3639-259e-4cec-b58f-b953c751e1cc)

**Comment by alltheseas** [comment]
12.9 GB reduced to 2.75 GB immediately upon clear cache

**Comment by alltheseas** [comment]
@jb55 

>  cleared my cache and it went from 21gb to 5gb. maybe unoptimized pfps ?

**Comment by alltheseas** [comment]
cleared cache from 16.97 GB -> 5.99 GB

@jb55 suggested nostrdb is remainder 

cc @danieldaquino
